conf/application.conf

```diff
@@ -26,6 +26,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 play.modules.enabled += "config.Module"
 play.modules.enabled += "com.dmanchester.playfop.sapi.PlayFopModule"
+play.modules.enabled += "config.DummyTADControllerModules"
```

Comment:
Should this be overridden in higher environments?

Reply:
Not necessarily as there is no functionality for TAD as yet. This can go all the way through as it will just return a blank PDF. when the TAD story is played this will be removed